1-starter-app/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "msdocs-azure-data-tables-sdk-js",
```

Comment:
We have to make sure that the package name we use here is owned by Microsoft, otherwise, we'll be vulnerable to some malicious developer taking the name in NPM and adding a malicious package. Usually, we have been using a scope like `@msinternal/<sample-package-name>`

Reply:
Update to `@msinternal/@azure/data-tables`? I'm not really sure.

Reply:
@joheredi , I don't know how I should update it here. Update to `@msinternal/@azure/data-tables`?

Reply:
not the tables sdk name but the sample project name `@msinternal/msdocs-azure-data-tables-sdk-js`